src/components/TaskBoard/TaskBoard.jsx

```diff
@@ -0,0 +1,30 @@
+import { useState } from "react";
+import TaskList from "@components/TaskList";
+import "./TaskBoard.scss";
+import { Button } from "@components/ui/form";
+
+function TaskBoard() {
+    const [isTaskCreating, setIsTaskCreating] = useState(false);
+
+    const toggleCreation = () => {
+        setIsTaskCreating((prevIsTaskCreating) => !prevIsTaskCreating);
+    };
+
+    return (
+        <section className="task-board">
+            <h1 className="task-board__heading">Add Tasks</h1>
+            <div className="task-board__action-wrapper">
+                <Button onClick={toggleCreation}>Create</Button>
+                <div className="task-board__filter-buttons">
+                    <Button>all</Button>
```

Comment:
```suggestion
                    <Button>All</Button>
```

Reply:
@ray-wtag bhaia, I've kept the text all lower case intentionally so that I can control it using CSS. And I've CSS property `capitalize` here. Please, suggest me what is the best practice here...

Reply:
it seems like you are taking the long road. It doesn't need any CSS, where you can just write `All` instead of `all`.